### PR TITLE
LINK-1996 | Move signups to attending on attendee capacity increase

### PR DESCRIPTION
**Pull request by harriris-vincit**
### Description
When a registration's maximum attendee capacity is increased, move as many waitlisted signups to attendees as possible.
### Closes
[LINK-1996](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1996)

[LINK-1996]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1996?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr912.api.dev.hel.ninja** :rocket::rocket::rocket:

File: registrations/models.py [hunk 1]
```diff
@@ -531,17 +531,20 @@ def calculate_remaining_waiting_list_capacity(self):
             waiting_list_capacity - waiting_list_count - reserved_seats_amount, 0
         )
 
-    def get_first_waitlisted(self):
+    def get_n_waitlisted(self, n_count: int):
```
Comment (nicobav) [inline registrations/models.py:534]: I'd prefere just:

```suggestion
    def get_waitlisted(self, count: int):
        """Gets given count of wait-listed attendees."""
```

File: registrations/utils.py [hunk 2]
```diff
@@ -151,16 +151,20 @@ def strip_trailing_zeroes_from_decimal(value: Decimal):
     return value.normalize()
 
 
-def move_first_waitlisted_to_attending(signup_or_signup_group):
-    registration = signup_or_signup_group.registration
-
-    if (
+def move_n_waitlisted_to_attending(registration, n_count: int):
```
Comment (nicobav) [inline registrations/utils.py:154]: Same here, I'd prefere something like:
```suggestion
def move_waitlisted_to_attending(registration, count: int):
    """Changes given number of wait-listed attendees in registration to be attending."""
```

**Review by nicobav — APPROVED** [review]
LGTM :shipit: 

Added some minor readability tunings for you to consider.

File: registrations/models.py (hunk 1 above)
Comment (harriris-vincit) [reply registrations/models.py]: Changed as suggested,

File: registrations/utils.py (hunk 2 above)
Comment (harriris-vincit) [reply registrations/utils.py]: Changed as suggested (same applies to all occurrences of this function and the `get_waitlisted` method).

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr912.api.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**